lib/main.js

```diff
@@ -344,7 +344,7 @@ MjPageJob.prototype.run = function() {
                     styles.appendChild(document.createTextNode(result.css));
                     document.head.appendChild(styles);
                 }
-                if (typesetConfig.useGlobalCache) {
+                if (result.svg && typesetConfig.useGlobalCache) {
```

Comment:
I'd reverse the order to fail early -- `result.svg` maybe be truthy when `useGlobalCache` is false.